### ISSUE TITLE
Can we load source file in the load function

### ISSUE THREAD

**Issue by puneetsl** [aeroxis/sultan#44]
Right now I am loading the context with some source which contains informtion about paths etc.
```python
with Sultan.load(user='puneet', logging=False, hostname="hadoopbox",
                 cwd="/home/user/puneet && source /home/user/puneet/hadoop.env") as s:
    for x in s.hadoop('fs -ls /user/puneet').pipe().sed("-r \"s/ +/ /g\"").pipe().cut("-d\" \" -f8").run():
        print(x)
```

I think this can be made better by having an optional parameter which loads source while loading the context. Or, maybe it's already there and I am missing something. 

Something like this would work great
```python
with Sultan.load(user='puneet', logging=False, hostname="hadoopbox",
                 cwd="/home/user/puneet", source="/home/user/puneet/hadoop.env") as s:
    for x in s.hadoop('fs -ls /user/puneet').pipe().sed("-r \"s/ +/ /g\"").pipe().cut("-d\" \" -f8").run():
        print(x)
```

**Comment by puneetsl** [comment]
If this looks okay to you I can create a pull request @davydany

**Comment by davydany** [comment]
It's implemented now in `Sultan v0.6`. Here are the docs, just like how you wanted it: http://sultan.readthedocs.io/en/latest/sultan-examples.html#example-11-sourcing-a-file-before-running-a-command

**Comment by davydany** [comment]
Please reopen the issue if this is still an issue.
